projects/clr/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Full documentation for HIP is available at [rocm.docs.amd.com](https://rocm.docs.amd.com/projects/HIP/en/latest/index.html)
 
+## HIP 7.2 for ROCm 7.2
+
+### Optimized
+* Removed Perl - package level dependency. 
+* roc-obj* - are now removed from hip and migrated to llvm. Refer to llvm-objdump for these tools.
```

Comment:
"are now removed from HIP repos/pkgs and equivalent functionality is provided by llvm-* tools, which are part of the llvm-project repo / the rocm-llvm-dev pkg."

@kzhuravl - please review the suggested modification. Also, is there a doc that explains how to map roc-obj* cmds to llvm-* cmds? This would help users transition from one to the other.